src/helm/common/hierarchical_logger.py

```diff
@@ -26,14 +26,18 @@ def __init__(self) -> None:
     def indent(self) -> str:
         return "  " * len(self.start_times)
 
-    def track_begin(self, x: Any) -> None:
-        print(self.indent() + str(x) + " {")
+    def track_begin(self, x: Any, print_bracket: bool = True) -> None:
```

Comment:
Why do we want to disable the bracket / time?  If it's just cosmetic, not sure it's worth adding the complexity...

Reply:
It's just that I am using this for printing the stack trace, and with the brackets and time, it is kind of unreadable. If you prefer, I could not use the `htrack_block` and use `\t` in my logs instead.

Reply:
Agreed, I think we can just revert this file.

Reply:
Reverted